.packit.yaml

```diff
@@ -0,0 +1,14 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+upstream_package_name: podman
+downstream_package_name: podman
+
```

Comment:
Packit doesn't have any capability right now to obtain spec files. Hence the error about not finding it. I can see how it could be a useful feature to copy it from Fedora when downstream_package_name is specified.

You can actually do that yourself: get the spec file from Fedora just to get an RPM build & not include it in the upstream git repo.

something like this should solve the problem:
```
actions:
  post-upstream-clone:
  - "curl -O https://src.fedoraproject.org/rpms/podman/raw/main/f/podman.spec"

Reply:
`production-build:f36 — RPM build succeeded.`

Happy to see that worked :)

Reply:
@TomasTomecek yes!!! btw, is there a convenient way to check the srpm files including spec file modification used for the scratch builds and/or make it downloadable perhaps? If it's already documented, a pointer link would be highly appreciated. Thanks a lot for the help on this!

Reply:
ah found the web logs url